code/datums/components/beetlejuice.dm

```diff
@@ -52,6 +52,9 @@
 			count -= speaker
 			apport(speaker)
 
+/datum/component/beetlejuice/UnregisterFromParent()
+	. = ..()
+	UnregisterSignal(SSdcs,COMSIG_GLOB_LIVING_SAY_SPECIAL)
```

Comment:
If you're going to use this proc move the signal register into RegisterWithParent()

Also you don't need to call parent in this proc, this exists purely for registering signals for components

Reply:
I don't really care about moving between objects, so i'll just move this to destroy.